testinfra/modules/file.py

```diff
@@ -61,7 +61,7 @@ def linked_to(self):
         >>> host.file("/var/lock").linked_to
         '/run/lock'
         """
-        return self.check_output("readlink -f %s", self.path)
+        return self.check_output("realpath %s", self.path)
```

Comment:
I wonder what the status of availability of realpath and readlink binaries in various linux/bsd distros...

Maybe we could try one and then another.